simplesource-command-kafka/src/main/java/io/simplesource/kafka/internal/streams/EventSourcedTopology.java

```diff
@@ -148,6 +152,16 @@ private void publishAggregateUpdates(final KStream<K, AggregateUpdateResult<A>>
         aggregateStream.to(topicName(aggregate), aggregatedUpdateProduced);
     }
 
+    private  void publishCommandResponse(final KStream<K, AggregateUpdateResult<A>> aggregateUpdateStream) {
+        final KStream<UUID, CommandResponse> aggregateStream = aggregateUpdateStream
+                .map((key, update) -> new KeyValue<>(
+                        update.commandId(),
+                        new CommandResponse(update.commandId(), update.readSequence(), update.updatedAggregateResult().map(x -> x.sequence()))
```

Comment:
Why do you store `commandId` in the `CommandResponse` while the topic is already keyed by the same field?

Reply:
I think I'm going to switch the key type to `K` (the aggregate key type).